src/System.Reflection.Metadata/src/System/Reflection/Metadata/PortablePdb/MethodBody.cs

```diff
@@ -37,5 +36,16 @@ public BlobHandle SequencePoints
                 return _reader.MethodBodyTable.GetSequencePoints(Handle);
             }
         }
+
+        /// <summary>
+        /// Returns local signature handle.
+        /// </summary>
+        public StandaloneSignatureHandle LocalSignature
+        {
+            get
+            {
+                return StandaloneSignatureHandle.FromRowId(_reader.GetBlobReader(SequencePoints).ReadCompressedInteger());
+            }
+        }
```

Comment:
For the type: MethodBodyDebugInfo? MethodDebugInfo?
For the SequencePoints property, maybe just Blob?